muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/user-panel.tsx

```diff
@@ -114,28 +115,30 @@ export default class UserPanel extends React.Component<
                     lastName: user.lastName,
                     email: user.email,
                     id: user.id,
-                    role: user.role ? user.role : "STUDENT",
+                    roles: user.roles ? user.roles : [Role.Student],
```

Comment:
Unnecessary null/undefined check as roles is always empty list by default. Can use includes checking straight